src/game/models/mod.rs

```diff
@@ -36,6 +36,16 @@ pub trait Positioned {
     }
 
     fn pos(&self) -> vector::Vector;
+
+    fn dx(&self) -> f64 {
```

Comment:
That's a good idea. I'll read up about the inheritance. If we rename it to Kinetic, probably wouldn't need to inherit, right? So, the implementation that I have right now didn't end up using the gel stuff. Could even take it out for now.